src/Compilers/CSharp/Portable/Symbols/Source/SourceNamespaceSymbol.cs

```diff
@@ -99,6 +99,19 @@ public override ImmutableArray<Location> Locations
             }
         }
```

Comment:
Is there no need for this to implement TryGetFirstSymbol, or was that done in a different PR?